packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -3218,6 +3218,8 @@ class SizedOverflowBox extends SingleChildRenderObjectWidget {
 ///    subtly).
 ///  * [TickerMode], which can be used to disable animations in a subtree.
 ///  * The [catalog of layout widgets](https://flutter.dev/widgets/layout/).
+///  * [Element.debugVisitOnstageChildren], for non-offstage children
```

Comment:
It isn't clear here why this is being linked. Is this something users would want to use in a test? Why would onstage children be important here?

Reply:
Because "offstage" is a term in Flutter. But when I search "flutter offstage", I only get `Offstage` widget, and then I think Offstage *widget*'s definition is *all* about "offstage". However, we know that its real definition is in debugVisitOnstageChildren's doc, which is broader.

P.S. searching result

![image](https://user-images.githubusercontent.com/5236035/190828951-d969a20b-abe1-4ef2-8954-817e95d36c80.png)

Reply:
Thanks! I was referring to the context on which a developer reading the api docs would understand this reference. What does this mean to the user? Can you add more context? When would they use it?

Reply:
Thanks for the reply, I will go back to this pr later

Reply:
@fzyzcjy will you be returning to this PR? Or should we close it for now?